lib/netsuite/support/records.rb

```diff
@@ -49,7 +49,15 @@ def to_attributes!(hash, kname, v)
       end
 
       def record_type
-        "#{record_namespace}:#{self.class.to_s.split('::').last}"
+        "#{record_namespace}:#{record_type_without_namespace}"
+      end
+
+      def netsuite_type
+        record_type_without_namespace.lower_camelcase
+      end
+
+      def record_type_without_namespace
+        "#{self.class.to_s.split('::').last}"
```

Comment:
Yes!! I've been wanting to get something like this in the codebase for some time!